boards/sim/sim/sim/configs/usrsocktest/defconfig

```diff
@@ -13,6 +13,8 @@ CONFIG_ARCH_CHIP="sim"
 CONFIG_ARCH_SIM=y
 CONFIG_BOARDCTL_POWEROFF=y
 CONFIG_BUILTIN=y
+CONFIG_DEBUG_ASSERTIONS=y
+CONFIG_DEBUG_FEATURES=y
 CONFIG_DEBUG_SYMBOLS=y
```

Comment:
@xiaoxiang781216 
Should we add these configs to `./boards/sim/sim/sim/configs/citest/defconfig` instead of this file?

Reply:
let me create a new patch for ./boards/sim/sim/sim/configs/citest/defconfig.

Reply:
Here is the patch: https://github.com/apache/nuttx/pull/8364